src/genericComponents/PrivateLabel.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Badge as styledBadge } from 'react-bootstrap';
+import styled from 'styled-components';
+import { colors, fonts } from '../theme/styleVars';
+
+const PrivateLabel = () => {
+  return <CAGLabel>private</CAGLabel>;
+};
+
+const CAGLabel = styled(styledBadge)`
+  border-radius: 7px;
+  font-family: ${fonts.montserrat};
+  color: ${colors.secondaryFontColor};
+  letter-spacing: 0.15rem;
+  font-weight: lighter;
+  padding: 8px 10px;
+  text-transform: uppercase;
+  background-color: grayishLightGreen;
```

Comment:
This would need to be used as a variable (property on the colors object) in the es6 string, so:
```
backround-color: ${colors.grayishLightGreen};
```

Reply:
Ah, right! Sorry for missing these minor details! Will change that :^)